tests/cephadm/test_cephadm_operations_via_admin.py

```diff
@@ -44,9 +45,12 @@ def run(ceph_cluster, **kw):
             f"Failed to add label '{label}' on node '{hostname}'"
         )
     # Verify files in new admin node
-    for file in files:
-        result = os.path.exists(file)
-        if result is False:
+    timeout, interval = 300, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        for file in files:
+            if os.path.exists(file):
```

Comment:
Check is for all files ? this will break the loop even single file from `files` is present.

Reply:
Changed to check for all files (two)